sphinx_js/typedoc.py

```diff
@@ -138,7 +138,7 @@ def make_type_name(self, type):
             names = ['[' + ','.join(types) + ']']
         elif type.get('type') == 'union':
             types = [self.make_type_name(t) for t in type.get('types')]
-            names = [' | '.join(types)]
+            return types
```

Comment:
I think this whole method is a bit weird, why make single item lists and then convert them back to a string... Do you want to refactor it? If not I can give it a try in another PR.

Reply:
Should be done with commit 31c4aba, please have a look again.